src/views/shared/private-layout/private-layout.view.tsx

```diff
@@ -1,8 +1,12 @@
 import React from "react";
 
-import usePrivateLayoutStyles from "./private-layout.styles";
+import usePrivateLayoutStyles from "src/views/shared/private-layout/private-layout.styles";
 
-function PrivateLayout({ children }) {
+interface PrivateLayoutProps {
+  children: JSX.Element;
+}
+
+function PrivateLayout({ children }: PrivateLayoutProps): JSX.Element {
```

Comment:
I would suggest to declare it as:

```ts
const PrivateLayout: React.FC = ({ children }) => { //... }
```

Doing this you are going to inherit the `children` from the React interface, which I think is better.

Reply:
Ok. Done.